android/MobiledgeXSDKDemo/app/src/main/AndroidManifest.xml

```diff
@@ -20,7 +20,7 @@
 
     <application
         android:usesCleartextTraffic="true"
-        android:allowBackup="true"
+        android:allowBackup="false"

```

Comment:
With this as "true", settings/permissions were persisting even after a full uninstall, making it difficult to test a first-install test case.

Reply:
Wonder if there's a runtime switch for this...